### ISSUE TITLE
modification of the XSLT for HVD

### ISSUE THREAD

**Issue by benoitdavidfr** [SEMICeu/iso-19139-to-dcat-ap#45]
An extension to DCAT-AP will be produced by the EC for HVD by the end of 2023 [DCAT-AP for HVD](https://semiceu.github.io/DCAT-AP/releases/3.0.0-hvd/).
Do you plan to modify the XSLT so that ISO 19139 MD can be transformed to a DCAT-AP MD that conforms to this new extension ?

**Comment by jakubklimek** [comment]
Yes, this update is ongoing now. Since INSPIRE is AFAIK not extending the input metadata record for HVD, the XSLT can include only what can be mapped automatically based on the current metadata. This means:
1. INSPIRE themes to HVD categories for datasets and data services
2. tagging of mapped datasets, services and distributions with the appropriate dcatap:applicableLegislation

However, since HVD is about more than just including these additional metadata items, the question is whether doing this automatically for all datasets is a good idea or whether this should be made somehow optional.

**Comment by jakubklimek** [comment]
Update: INSPIRE MIG-T has collected evidence of good practice of how to identify HVD in INSPIRE metadata. This is still to be officially formulated. When that happens, it can be implemented in the XSLT.
